template/_dynamic_files.jinja2

```diff
@@ -180,6 +188,28 @@ imports/%_import.owl: mirror/%.owl
 	echo "ERROR: You have configured your default module type to be custom; this behavior needs to be overwritten in {{ project.id }}.Makefile!" && touch $@
 {% endif %}
 {%- endif %}
+{% if project.import_group.use_base_merging %}
```

Comment:
This reveals another mistakes, which is that the initial ODK sets up imports in RDFXML format, while the default format for the respective ROBOT pipeline is functional syntax..

Reply:
It does not really matter, though, does it? The placeholder import module will be replaced by a ROBOT-generated module in the expected functional syntax.

Reply:
If you are a fetishist of git history cleanliness, then the very first commit (which is automatically done by the ODK) will be an RDF XML file, and the second one a functional syntax file. Its not a big deal, just saying..

Reply:
(In any case, I am happy with this PR, feel free to merge and either make follow up issues on the OBO and syntax stuff, or leave it!)

Reply:
You know my position about how a PR should be about only one problem at a time :) so I’ll merge this one now and make a new one to fix the existing inconsistencies.